common/views/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+
+import React from 'react';
+import styled from 'styled-components';
+import { classNames } from '../../../utils/classnames';
+import Space from '../styled/Space';
+
+const CheckboxLabel = styled.label.attrs({
+  className: classNames({
+    'flex flex--v-center': true,
+  }),
+})``;
+
+const CheckboxBox = styled.span.attrs({
```

Comment:
`Container`?

Reply:
❓

Reply:
This is the visible styled 'box' part of the checkbox, not a container.

i.e. this _thing_
![image](https://user-images.githubusercontent.com/1394592/66565351-28774580-eb5a-11e9-8e02-0a2744c7be64.png)